views/index.hbs

```diff
@@ -0,0 +1,33 @@
+<header class="jumbotron">
+  <div class="container">
+    <h1>Conformance Status Page</h1>
+    <p>Introduction text</p>
+  </div>
+</header>
+
+
+<section class="container">
+  <p>Main body section</p>
+  <table class="table table-hover datasets-table">
+    <thead>
+    <tr>
+      <th scope="col">col1</th>
+      <th scope="col">col2</th>
+      <th scope="col">col3</th>
+    </tr>
+    </thead>
+    <tbody>
+    {{#each statusData.publishers}}
+      <tr>
+        <td>{{@index}}</td><td>{{this.name}}</td><td>{{this.url}}</td>
```

Comment:
I suspect they won't want numbers there, or if they do, they will want ones that stay same and might not change over time at least. Users associate all kinds of meaning to numbers that was never intended. But again, don't think this has to be a blocker.